Node.js/test/basicget.test.js

```diff
@@ -0,0 +1,234 @@
+    let credentials;
+    let od;
+    let result;
+
+    beforeEach(() => {
+        MQDetails = {};
+        credentials = {};
+        cno = new mq.MQCNO();
+    });
+
+    it('Should perform MQOPEN', async () => {
+        for (let i = 0; i < envConfigLength; i++) {
+            await buildMQDetails(MQDetails, credentials, i);
+            await initialise(cno, MQDetails, credentials);
+            result = await connx(cno, MQDetails);
+            od = await open(result, MQDetails);
+            expect(od._name).to.equal(envConfig.MQ_ENDPOINTS[i].QUEUE_NAME);
+            expect(od._mqQueueManager._name).to.equal(envConfig.MQ_ENDPOINTS[i].QMGR);
+            expect(od._hObj).to.equal(101);
+        }
+    }).timeout(5000);
+});
+
+describe('close function', () => {
+    let cno;
+    let MQDetails;
+    let credentials;
+    let result;
+    let od;
+
+    beforeEach(() => {
+        MQDetails = {};
+        credentials = {};
+        cno = new mq.MQCNO();
+    });
+
+    it('Should perform MQCLOSE', async () => {
+        for (let i = 0; i < envConfigLength; i++) {
+            await buildMQDetails(MQDetails, credentials, i);
+            await initialise(cno, MQDetails, credentials);
+            result = await connx(cno, MQDetails);
+            od = await open(result, MQDetails);
+            await close(od, i);
+            assert.equal(true, true);
+        }
+    }).timeout(5000);
+});
+
+describe('disc function', () => {
+    let cno;
+    let MQDetails;
+    let credentials;
+    let result;
+
+    beforeEach(() => {
+        MQDetails = {};
+        credentials = {};
+        cno = new mq.MQCNO();
+    });
+
+    it('Should perform MQDISC', async () => {
+        for (let i = 0; i < envConfigLength; i++) {
+            await buildMQDetails(MQDetails, credentials, i);
+            await initialise(cno, MQDetails, credentials);
+            result = await connx(cno, MQDetails);
+            await open(result, MQDetails);
+            await disconnect(result, i);
+            assert.equal(true, true);
+        }
+    }).timeout(5000);
+});
+
+describe('getMessage function', async () => {
+    let cno;
+    let MQDetails;
+    let credentials;
+    let result;
+    let od;
+    let openOptions;
+    let msgObject;
+    let msg;
+    let mqmd;
+    let pmo;
+    let rcvMsg;
+
+    beforeEach(() => {
+        MQDetails = {};
+        credentials = {};
+        cno = new mq.MQCNO();
+        od = new mq.MQOD();
+        mqmd = new mq.MQMD();
+        pmo = new mq.MQPMO();
+        openOptions = MQC.MQOO_OUTPUT;
+        pmo.Options = MQC.MQPMO_NO_SYNCPOINT | MQC.MQPMO_NEW_MSG_ID | MQC.MQPMO_NEW_CORREL_ID;
+        mqmd.Persistence = MQC.MQPER_PERSISTENT;
+        msgObject = {
+            "Key": "0"
+        }        
+        msg = JSON.stringify(msgObject);
+
+    });
+
+    it('Should perform MQGET', async () => {
+        for (let i = 0; i < envConfigLength; i++) {
+            await buildMQDetails(MQDetails, credentials, i);
```

Comment:
This is a copy and paste of the put logic. Instead, just run `basicput`, then test that the get functions run.